backend/index.ts

```diff
@@ -35,3 +36,53 @@ const port = 8000;
 app.listen(8000, () => {
     console.log(`Server started: http://localhost:${port}/`);
 }); 
+
+
+
+// Configure Strategy for Login
+
+// user is prompted to sign in with their username and password by rendering a form
+// defining a route
```

Comment:
love that you use comments to guide your work, but I don't think these are necessary to keep here!

Reply:
Will remove rn!